### DISCUSSION TITLE
Suggested indexes for Fast Data

### DISCUSSION THREAD

**Discussion by fabio-percivaldi** [mia-platform/community#91] [Q&A] [answered]
Low Code Fast Data is doing a lot of not so clear operation on the projections, projections changes and single views and it's not always easy to understand which indexes should be created. Is it correct to assume that a part of the fields of the indexes are fixed and they do not depend on the System of Record, for example the projections changes query performed by the Single View Creator is always performed on the same fields (type, changes.state etc)

Is this assumption correct? if so can we create a page on the documentation with the suggested indexes that a user should create when setupping Fast Data?

**Comment by TheRusCompass** [comment]
Yes, the assumption is correct. That's a good idea to suggest the proper indexes in documentation

**Reply by TheRusCompass** [reply, marked as answer]
It is possible to find the index suggestion in [this documentation page](https://docs.mia-platform.eu/docs/fast_data/configuration/indexes).
